azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -32,6 +32,9 @@ void uploadExecutableFlow(ExecutableFlow flow)
   ExecutableFlow fetchExecutableFlow(int execId)
       throws ExecutorManagerException;
 
+  List<ExecutableFlow> fetchRecentlyFinishedFlows(long lifeTimeMs)
```

Comment:
It's not obvious what the lifeTimeMs means. Add a javadoc?

Does the name like "maxAge" more closely reflect the intent?

How about using the Duration instead here and convert to long when needed? At this level of abstraction, users of this API shouldn't share how this value is represented in the DB.